### ISSUE TITLE
Can't download TwiLightMenu-Updater

### ISSUE THREAD

**Issue by ccfman2004** [RocketRobz/TWiLightMenu-Updater#113]
The TwiLight-Updater can no longer download updates to itself anymore.

**Comment by RocketRobz** [comment]
Is there an error message?

**Comment by ccfman2004** [comment]
No.  It tries to down this 1 byte file and then goes back to the main menu.

**Comment by ccfman2004** [comment]
I am wondering if the issue is that you changed the file name of the downloaded file and the 4.4.0 and earlier is trying to download the original named file or is there something else that is causing it.  I have 2 N3/2DS XLs with 4.4.0 cia and they both exhibit the same issue.

I'll try the 3dsx file to see if I have the same issue.
